test/integration/targets/azure_rm_virtualmachine/inventory/hosts.yml

```diff
@@ -0,0 +1,5 @@
+all:
+  hosts:
+    azure_testhost_1:
+    azure_testhost_2:
+    azure_testhost_3:
```

Comment:
If possible, use a more descriptive name for each host that describes what is being tested.

Reply:
I gave the hosts more descriptive names based on the tests being run.